src/rust/engine/testutil/mock/src/execution_server.rs

```diff
@@ -139,11 +140,11 @@ impl MockResponder {
   }
 
   fn log<T: protobuf::Message + Sized>(&self, message: T) {
-    self
-      .received_messages
-      .lock()
-      .unwrap()
-      .push((message.descriptor().name().to_string(), Box::new(message)));
+    self.received_messages.lock().unwrap().push((
+      message.descriptor().name().to_string(),
+      Box::new(message),
+      Instant::now(),
+    ));
```

Comment:
no change here, just fmt. somehow the formatter doesn't like the existing format :/